__tests__/index.spec.ts

```diff
@@ -1,30 +1,48 @@
 import * as https from 'https'
 
-import { expect } from 'chai'
+import { Assertion, expect } from 'chai'
 
 import { defaultRules, getNews, LodestoneNews, regions } from '../src'
 
 const UTC_0_2022_07_19_07_59_59 = 1658217599 * 1000 // 2022-07-19 07:59:59 UTC+0
 
+const a = function (_super: typeof Assertion) {
+  return function (this: typeof Assertion, value: any, message?: string) {
+    if (value === 'news') {
+      new Assertion(this._obj).to.have.property('title').which.is.a('string')
+      new Assertion(this._obj).to.have.property('url').which.is.a('string')
+      new Assertion(this._obj).to.have.property('epoch').which.is.a('number')
+      new Assertion(this._obj).has.property('date').which.is.a('date')
+    } else {
+      _super.apply(this, [value, message])
+    }
+  }
+}
+
+function noop() {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  return function () {}
+}
+
+Assertion.overwriteChainableMethod('a', a, noop)
+Assertion.overwriteChainableMethod('an', a, noop)
+
 regions.forEach((region) => {
+  const categories = ['topics', 'notices', 'maintenance', 'updates', 'status']
+
   describe(`Locale: ${region}`, () => {
-    ;['topics', 'notices', 'maintenance', 'updates', 'status'].forEach(
-      (category) => {
-        it(`should fetch ${category}`, async () => {
-          const news = await getNews({
-            region,
-            category,
-            count: 20,
-          })
-          expect(news).is.an('array')
-          expect(news).has.length.greaterThan(0)
-          expect(news[0]).has.property('title').which.is.a('string')
-          expect(news[0]).has.property('epoch').which.is.a('number')
-          expect(news[0]).has.property('url').which.is.a('string')
-          expect(news[0]).has.property('date').which.is.a('date')
-        }).timeout(0)
-      },
-    )
+    categories.forEach((category) => {
+      it(`should fetch ${category}`, async () => {
+        const news = await getNews({
+          region,
+          category,
+          count: 20,
+        })
+        expect(news).is.an('array')
+        expect(news).has.length.greaterThan(0)
+        expect(news[0]).is.a('news')
```

Comment:
现在可以这么用，`haveLen()` 入参为空则判断大于0
```suggestion
        expect(news).is.a.newsList.that.haveLen()
```